File: Data Structures & Algorithms/two-integer-sum/submission-1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        map<int, int> store;
        vector<int> res;
        int n = nums.size();
        for(int i=0; i<n; i++){
            if(store.contains(target-nums[i])){
                res.push_back(store[target-nums[i]]);
                res.push_back(i);
            }
            else{
                store[nums[i]] = i;
            }
        }
        return res;
    }
};

File: Data Structures & Algorithms/remove-element/submission-1.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        left = 0
        right = len(nums)-1
        while(left<=right):
            if(nums[right] == val):
                right-=1
                continue
            if(nums[left] == val):
                nums[right], nums[left] = nums[left], nums[right]
                left+=1
                right-=1
            elif(nums[left]!=val):
                left+=1
        return left